scripts/test

```diff
@@ -6,9 +6,14 @@ realpath() {
     [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
 }
 
+if [ $# -gt 0 ] ; then
```

Comment:
That change does not seems to be related to timezone..

Reply:
No, it is not. It's just a little addition I had in that script to limit the number of concurrent runs. I noticed when I created the pull request. Should be of no harm... ;-)